docs/ref/models/fields.txt

```diff
@@ -492,7 +492,8 @@ optional arguments:
     it's not just a default value that you can override. So even if you
     set a value for this field when creating the object, it will be ignored.
     If you want to be able to modify this field, set ``default=timezone.now``
-    (from :func:`django.utils.timezone.now`) instead of ``auto_now_add=True``.
+    (from :func:`django.utils.timezone.now`) or ``default=date.today``
```

Comment:
worth making it explicit: "timezone.now for DateTimeField or date.today for DateField"?